test/test_partial.py

```diff
@@ -137,6 +137,7 @@ def test_partial_wasserstein():
 
 
 def test_partial_gromov_wasserstein():
+    np.random.seed(42)
```

Comment:
@eloitanguy @rflamary you should not use global seeds in tests. It makes tests coupled as they share a random state. It may happen that a test that does not control it's seed fails only if run after another test and it makes debugging a nightmare. You should np.random.default_rng these days.